src/cerbos-compile-and-test.ts

```diff
@@ -29,32 +28,35 @@ async function cerbosCompileAndTest(
   }
 
   core.info(`Command to run: ${command}`)
-
   core.startGroup('cerbos compile results')
-  let stdout = ''
-  try {
-    stdout = child.execSync(command, {
-      encoding: 'utf8'
-    })
-  } catch (error) {
-    const execSyncError = asExecSyncException(error)
-
-    switch (execSyncError.status) {
-      case 1: // returns 1 if there are compilation errors
-        core.setFailed(`Compilation errors detected: ${error}`)
-        break
-      default:
-        core.setFailed(`Failed to launch Cerbos: ${error}`)
-        break
+  child.exec(command, (err, stdout, stderr) => {
+    if (err || stderr) {
+      core.setFailed(`Error(s) occurred`)
+      if (err) {
+        core.error(err.message)
+      }
+      if (stderr) {
+        core.error(stderr)
+      }
+      if (stdout) {
+        core.info(
+          `${styles.default.color.ansi16m(
+            ...styles.default.hexToRgb('#00ff00')
+          )}${stdout}${styles.default.color.close}`
+        )
```

Comment:
I think GH logging can handle coloured output (https://github.com/actions/toolkit/blob/main/packages/core/README.md#styling-output). Since `cerbos compile` has its own coloured output, do we need to do this here?